From 829542cd0c977bcc34e225e404ad725d586cdb0d Mon Sep 17 00:00:00 2001
From: Stefan Lippers-Hollmann <s.L-H@gmx.de>
Date: Sat, 23 Nov 2013 01:44:17 +0000
Subject: [PATCH 70/70] r92su: remove superfluous version.h include

"r92su: cfg80211_rx_mgmt api change" disables the offending code
unconditionally, avoiding the need for LINUX_VERSION_CODE macros.

Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>
---
 r92su/rx.c | 1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/wireless/r92su/rx.c
+++ b/drivers/net/wireless/r92su/rx.c
@@ -29,7 +29,6 @@
  *
  *****************************************************************************/
 #include <linux/kernel.h>
-#include <linux/version.h>
 #include <linux/etherdevice.h>
 
 #include <net/ieee80211_radiotap.h>
